src/TyroSharedContext.tsx

```diff
@@ -267,17 +271,20 @@ const TyroProvider = ({ children, options }: TyroPayContext): JSX.Element => {
         return;
       }
     }
-    try {
-      setTyroErrorMessage(null);
-      setIsSubmitting(true);
-      await submitPayRequest(paySecret, cardDetails, options[TyroPayOptionsKeys.liveMode]);
-    } catch (error) {
-      setTyroErrorMessage(errorMessage(TyroErrorMessages[ErrorMessageType.PAYMENT_FAILED]));
-      setIsSubmitting(false);
-      return;
-    }
+    setTyroErrorMessage(null);
+    setIsSubmitting(true);
+    let submissionError;
+    await submitPayRequest(paySecret, cardDetails, options[TyroPayOptionsKeys.liveMode]).catch((error) => {
```

Comment:
any reason we aren't using a try-catch here?

Reply:
changed to try catch